pandas/core/ops/docstrings.py

```diff
@@ -233,6 +233,60 @@ def _make_flex_doc(op_name, typ):
 dtype: float64
 """
 
+_eq_example_SERIES = """
+Examples
+--------
+s1 = pd.Series([1, 2, 3, 4, 5])
```

Comment:
Do these need leading `>>>`?

Reply:
I think these are needed here as well - the others in the module have them if you need a reference